guides/common/modules/proc_migrating-from-katello-agent-to-remote-execution.adoc

```diff
@@ -15,12 +15,12 @@ endif::[]
 * You have previously installed the `katello-agent` package on content hosts.
 
 .Procedure
-. If you have Remote Execution configured to use `pull-mqtt` mode: Deploy the remote execution pull client to the hosts.
-ifdef::katello[]
+. If you have Remote Execution configured to use `ssh` mode, distribute the remote execution SSH keys to the hosts.
+For more information, see xref:Distributing_SSH_Keys_for_Remote_Execution_{context}[].
+. If you have Remote Execution configured to use `pull-mqtt` mode, deploy the remote execution pull client to the hosts.
```

Comment:
Shouldn't we have a link here to the steps to do so?

Reply:
Yes, the link is located inside the ifdef below. The ifdef was there only for katello, but it should have been there for satellite and orcharhino as well, which I've corrected in this PR.

Reply:
aah, gotcha!